### PR TITLE
chore(components): Updated components to latest versions

### PR DESCRIPTION
**Pull request by lordelogos**
### Description
This PR contains the following:
- Updates to the following components: `body`, `button`,`font`, `html`
- Added the `@react-email/markdown` component

### CODE REVIEW

**Comment by lordelogos** [comment]
@bukinoshita, This PR will update the `font` component and other components that are not at the latest version in `@react-email/components` and will fix the issues with font from discord.

The new version of font (v0.0.3) solves the issue but was not updated in the components package.

After you merge this and release a new package version, it should be solved with an update of the `@react-email/components` package

**Comment by shellscape** [comment]
I've merged this in a soft-fork here https://github.com/shellscape/jsx-email/commit/976eb95f5c456c232ca09e76a3545ead26a105b2 and will publish this as `@jsx-email/components` (along with the other components) later this evening.

File: packages/components/package.json [hunk 1]
```diff
@@ -45,17 +45,18 @@
...
-    "@react-email/font": "0.0.2",
+    "@react-email/font": "0.0.3",
     "@react-email/head": "0.0.5",
     "@react-email/heading": "0.0.8",
     "@react-email/hr": "0.0.5",
-    "@react-email/html": "0.0.4",
+    "@react-email/html": "0.0.5",
     "@react-email/img": "0.0.5",
     "@react-email/link": "0.0.5",
+    "@react-email/markdown": "0.0.4",
```
Comment (vcapretz) [inline packages/components/package.json:59]: I believe all other packages ended up being updated already, but adding `markdown` can still be useful, so feel free to rebase to keep this change only 🙏

Comment (lordelogos) [reply packages/components/package.json]: Thanks for the update @vcapretz 

I've made the requested changes

**Pull request closed without merging**